src/ports/postgres/modules/deep_learning/madlib_keras.py_in

```diff
@@ -503,6 +525,21 @@ def internal_keras_evaluate(dependent_var, independent_var, model_architecture,
     independent_var = expand_input_dims(independent_var, target_type='float32')
     dependent_var = expand_input_dims(dependent_var)
 
+
     with K.tf.device(device_name):
         res = model.evaluate(independent_var, dependent_var)
+    if current_seg == -1:
```

Comment:
We should use `is_platform_pg()` instead of `current_seg` for this if check. It reflects the true intent of the if check